openedx/core/djangoapps/content/course_structures/models.py

```diff
@@ -1,83 +0,0 @@
-"""
-Django ORM model specifications for the Course Structures sub-application
-"""
-import json
-import logging
-
-from collections import OrderedDict
-from model_utils.models import TimeStampedModel
-from opaque_keys.edx.django.models import CourseKeyField, UsageKey
-
-from util.models import CompressedTextField
-
-
-logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
-
-
-class CourseStructure(TimeStampedModel):
```

Comment:
@edx/devops This PR removes the last usage of the `CourseStructure` model.  If we want to delete this SQL table from our environments, how should we do so?  As a Django migration?  Manually via SQL?  Or something else?
If via Django migration, should we keep the migrations directory and model around until the PR that drops the table. Thanks.